### ISSUE TITLE
How to use word_forms without internet access?

### ISSUE THREAD

**Issue by monocongo** [gutfeeling/word_forms#24]
I have inherited some code that uses `word_forms`. It needs to run on a system that has no internet access (firewalled). It looks to me that when `word_forms` initializes it calls `ntlk.download('wordnet')` which croaks after timing out. I wonder if there's a way for me to add the WordNet dataset locally and modify the `word_forms` code to have it initialize `nltk` with the local copy of WordNet instead?

For example:
```
>>> from word_forms.word_forms import get_word_forms
[nltk_data] Error loading wordnet: <urlopen error [WinError 10060] A
[nltk_data]     connection attempt failed because the connected party
[nltk_data]     did not properly respond after a period of time, or
[nltk_data]     established connection failed because connected host
[nltk_data]     has failed to respond>
>>> import nltk
>>> nltk.download('wordnet')
[nltk_data] Error loading wordnet: <urlopen error [WinError 10060] A
[nltk_data]     connection attempt failed because the connected party
[nltk_data]     did not properly respond after a period of time, or
[nltk_data]     established connection failed because connected host
[nltk_data]     has failed to respond>
False
```

Finally, thanks for all the effort that has gone into making this package available, it's very appreciated!

**Comment by tomaarsen** [comment]
Hello @monocongo 

If I am not mistaken, then the only place where `word_forms` requires internet access is the following lines:
https://github.com/gutfeeling/word_forms/blob/e096806ab5c0515ace83b303f799dbafc61aac37/word_forms/word_forms.py#L3-L8

This is, as you already noticed, for making sure the nltk wordnet corpus is downloaded. This is a one-time install. With other words - if you are able to access the internet once, you will be able to run the program once (or alternatively, simply run `import nltk; nltk.download("wordnet")`, after which `word_forms` *should* work offline.

Alternatively, if this is not an option, you can try to investigate to where NLTK stores its downloaded information. Conveniently, I'm a contributor to nltk as well, and have downloaded all corpuses they offer. As a result I was able to find the location of my corpuses after a bit of a search: I found a `nltk_data` folder in `C:\Users\<user>\AppData\Roaming` on Windows. A bit of googling shows that this same folder might be expected under `/usr/share/nltk_data` on Unix. 
My recommendation is to use `nltk.download("wordnet")` on your local machine with internet access, then figure out the path location and copy the files over to the system, in the exact same location. Assuming `nltk` only simply checks if the files are there or not (as opposed to like, setting a environment variable to state that the files are downloaded), you should be good to go.

Good luck!

If none of this works, you might also be able to find some help at https://github.com/nltk/nltk, but not that they don't reply to their issue very often or quickly.

**Comment by gutfeeling** [comment]
On my Ubuntu system, it seems that the `nltk_data` folder is in `~/nltk_data`.

[This link](https://www.nltk.org/data.html#manual-installation) has the official information on how to manually install these data packages. Might be helpful.

**Comment by monocongo** [comment]
I'm working on a Windows machine via remote desktop so I have downloaded the data locally and then uploaded it to the remote server via SFTP. The download was performed using the command `python -m nltk.downloader all`. Then in my Python code, I have added this:
```
os.environ['NLTK_DATA'] = 'H:/path/to/nltk_data'
```
Everything seems to work as advertised now.

I totally appreciate the fast, accurate help on this issue!
